docs/index.md

```diff
@@ -0,0 +1,191 @@
+---
+hide:
+  - navigation
+---
+
+![Merlin Banner](./assets/merlin_banner.png){ width="75%" }
+
+----
+
+# Merlin
+
+Empower your projects with Merlin, a cloud-based workflow manager designed to facilitate scalable and reproducible workflows, particularly suited for running many simulations and iterative procedures.
+
+[On GitHub :fontawesome-brands-github:](https://github.com/LLNL/merlin){: .md-button .md-button--primary }
+
+----
+
+## Why Merlin?
+
+Workflows, applications and machines are becoming more complex, but subject matter experts need to devote time and attention to their applications and often require fine command-line level control. Furthermore, they rarely have the time to devote to learning workflow systems.
+
+With the expansion of data-driven computing, the HPC scientist needs to be able to run more simulations through complex multi-component workflows.
+
+**Merlin targets HPC workflows that require many simulations**.
+
+## Getting Started
+
+Merlin can be installed via pip in your own virtual environment.
+
+1. First, create a virtual environment:
+
+    ```bash
+    python -m venv merlin_venv
+    ```
+
+2. Now activate the virtual environment:
+
+    ```bash
+    source merlin_venv/bin/activate
```

Comment:
there are a few times where we mention this. we should make a note this is for bash and for tcsh it'd be source ... activate.csh

Reply:
I'll add in a tabbed setting for this like I do for the "Launch in Parallel" and "Launch on the Current Node" options:

![image](https://github.com/LLNL/merlin/assets/49216024/dbd2eb24-3ed8-4ee5-a4cf-0cb39f878670)